catalog-rest-service/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -480,6 +480,10 @@
           "description": "No.of rows in the table. This is always executed on the whole table.",
           "type": "number"
         },
+        "columnNames": {
+          "description": "List of column names",
```

Comment:
Are these the columns on top of which the profiler is enabled? If so we should specify it in the description. Also, IDK if it might be easier to keep track of "disabled" columns instead 🤔 WDYT?

Reply:
Aaah I see, this is the new metric, LGTM

Reply:
@pmbrull yes it will tests that all the columns of the tables are there